api/generic-cti/hangup-event.rst

```diff
@@ -0,0 +1,215 @@
+Call hangup
```

Comment:
Proposal for the headline: Event: hangup

Reply:
What do you think, Gerrit?

Reply:
In case leave it as it is.

Reply:
I prefer "Call hangup", not because it provides a better headline for the article, but it provides a better bullet point for the menu on the left.

Reply:
Okay, we'll leave it as is then :)